src/mod/endpoints/mod_sofia/mod_sofia.c

```diff
@@ -1474,6 +1474,8 @@ static switch_status_t sofia_send_dtmf(switch_core_session_t *session, const swi
 	return SWITCH_STATUS_SUCCESS;
 }
 
+#define SIP_100REL_MIN_DIFF_PROGRESS_TIME 300
```

Comment:
This shouldn't be part of this PR

Reply:
I noticed it was added when I changed the based branch to development